includes/admin/class-wc-payments-admin.php

```diff
@@ -277,9 +280,42 @@ public function enqueue_payments_scripts() {
 		}
 
 		// TODO: Update conditions when ToS script is enqueued.
-		if ( false && $current_tab && $current_section
-		&& 'checkout' === $current_tab
-		&& 'woocommerce_payments' === $current_section ) {
+		$tos_agreement_declined = (
+			$current_tab
+			&& 'checkout' === $current_tab
+			&& isset( $_GET['tos-disabled'] ) // phpcs:ignore WordPress.Security.NonceVerification
+		);
+
+		$tos_agreement_required = (
+			$this->is_tos_agreement_required() &&
+			(
+				// Is settings page?
+				(
+					$current_tab
+					&& $current_section
+					&& 'checkout' === $current_tab
+					&& 'woocommerce_payments' === $current_section
+				)
```

Comment:
Non-blocker: but we do this check above anyway, so it's worth extracting into variable: `is_settings_page`.

Reply:
Done here: https://github.com/Automattic/woocommerce-payments/pull/993/commits/c078207901dfc0803352e2a2384d84db42bf43fe